lib/chef/application/windows_service.rb

```diff
@@ -264,6 +266,14 @@ def auto_log_level?
         Chef::Config[:log_level] == :auto
       end
 
+      def resolve_log_location
+        if Chef::Config[:log_location] && Chef::Config[:log_location].is_a?(String)
```

Comment:
It seems like in the old code we didn't pass -L at all if log_location was set to STDOUT, and now we're passing -L with DEFAULT_LOG_LOCATION if log_location is STDOUT?